.dependabot/config.yml

```diff
@@ -3,14 +3,17 @@ update_configs:
   - package_manager: "python"
     directory: "/"
     target_branch: "dev"
-    update_schedule: "live"
+    update_schedule: "weekly"
 
   - package_manager: "javascript"
     directory: "/"
     target_branch: "dev"
-    update_schedule: "live"
+    update_schedule: "weekly"
 
   - package_manager: "docker"
     directory: "/"
     target_branch: "dev"
     update_schedule: "weekly"
+
+    default_labels:
+      - "dependencies"
```

Comment:
This may actually have a negative effect (not tested), as the specific labels such as `python`, `javascript`, `docker` may go away if this overwrites dependabot default behavior.